### PR TITLE
Upgrading Plugin to Latest Gradle and Jasper

### PR DESCRIPTION
**Pull request by tkofford**
Upgrading the plugin to use a more recent version of gradle, v5.6.2, and
jasperreports, v6.9.0. As a result, moved away from the bintray
publishing plugin to instead use the gradle publishing plugin.

Moving to the gradle publishing plugin has several good & bad side
effects. The good was that it eliminated all the bintray tasks and
configuration and automatically generates the
src/main/resources/META-INF/gradle-plugins *.property file based on the
plugin id in the publishing plugin configuration. The bad is that the
author of the plugin will need to do a little work to use the "new" way
of publishing plugins to the gradle plugin portal as described in the
following link: https://plugins.gradle.org/docs/submit. Note that link
also has instructions at the top of the page on how to migrate
publishing from bintray.

Also updated version in gradle.properties to "0.4-SNAPSHOT".

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,57 +1,86 @@
...
-apply plugin: 'groovy'
-apply plugin: 'maven-publish'
-apply plugin: 'com.jfrog.bintray'
+// Unless overridden in the pluginBundle config DSL, the project version will
+// be used as your plugin version when publishing
+group = 'com.github.gmazelier'
+version = project.property('version')
 
 repositories {
+	mavenLocal()
```
Comment (ichromanrd) [inline build.gradle]: Does `mavenLocal` has to be included?

**Comment by ghost** [comment]
FYI: v6.10.0 of jasperreports seems to be the latest version. https://community.jaspersoft.com/project/jasperreports-library/releases

**Review by ghost — APPROVED** [review]

**Comment by tkofford** [comment]
Is there anything else that I need to do to get this pull request merged?

**Comment by gmazelier** [comment]
LGTM. Thanks for your contribution! I will check the Gradle portal documentation to publish the plugin in the next days.

**Comment by gmazelier** [comment]
New version released. Let me know if you have any issues.

**Comment by gmazelier** [comment]
Closes #13

**Pull request merged**